resources/assets/components/HistoryModal/index.js

```diff
@@ -36,7 +64,10 @@ class HistoryModal extends React.Component {
           </div>
 
           <h3>Reportback History</h3>
-          <p>table of all the history goes here 📖</p>
+          <p>A log of up to 20 of the most recent signup events! 📖</p>
```

Comment:
@ngjo does this copy make sense? We're going to hold on front end pagination to get more than 20 results (since via the API only returns 20 per page) so wanted to make that note here!

Reply:
hiyah @chloealee ! Can we change it to this:

Reply:
Yes that is so much better, thanks @ngjo ! Updated :) Looks like the below (still need to add user's name instead of NS ID which I'm assuming we want instead?):
![screen shot 2017-10-06 at 10 07 10 am](https://user-images.githubusercontent.com/9019452/31281705-37cd5f52-aa7e-11e7-85cb-a70d7c7adbff.png)

Reply:
@chloealee looking great! And @lkpttn what were you thinking for the User column? Name seems useful for scanning purposes but I could also see admin vs member indication and northstar ID being useful for searching too. 

Either way, we can tackle whatever Luke comes up with in a future (possibly next!) sprint as an enhancement!

Reply:
ok sounds good, thanks for the feedback, @ngjo ! So just leave as northstar id for now?